docs/workflows/react-native-specific-workflow-steps/react-native-ui-test.md

```diff
@@ -57,6 +58,17 @@ React Native UI Test component works according to the device given in the projec
 
 :::
 
+:::danger Step Rule
+
+There's one important setting that you should change. If any workflow steps fail, Appcircle automatically skips other steps and jumps directly to the Export Build Artifacts step. However, it is possible that some of your tests may fail. **If Test Report Component doesn't run, reports will not be generated.** You should turn on the following toggles so that Test Report Component always runs whether your tests fail or pass.
```

Comment:
I have two feedback for this warn:

1. Can you applied the my suggestion for this warn on Test Reports component document?
2. IMO this warn must be for **React Native UI Test** component, not for Test Reports

Already  Test Reports has own warn message.

Reply:
Same rule available for UI test, I update the sentence and the screenshot.